fog/types/src/view.rs

```diff
@@ -95,6 +97,39 @@ pub struct QueryResponse {
     pub last_known_block_cumulative_txo_count: u64,
 }
 
+/// Internal representation of the `MultiViewStoreQueryResponseStance` proto
+/// enum.
+#[derive(Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
+pub enum MultiViewStoreQueryResponseStatus {
+    /// The Fog View Store successfully fulfilled the request.
+    Success,
+    /// The Fog View Store is unable to decrypt a query within the
+    /// MultiViewStoreQuery. It needs to be authenticated by the router.
+    AuthenticationError,
+    /// The Fog View Store is not ready to service a MultiViewStoreQueryRequest.
+    /// This might be because the store has not loaded enough blocks yet.
+    NotReady,
+}
+
+/// Internal representation of the `MultiViewStoreQueryResponse` proto struct.
+#[derive(Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
+pub struct MultiViewStoreQueryResponse {
+    /// The encrypted QueryResponse.
+    pub encrypted_query_response: EnclaveMessage<NonceSession>,
+
+    /// The `ResponderId` for the store that fulfilled the request.
+    pub store_responder_id: ResponderId,
```

Comment:
I think that this makes sense, but we've discussed moving to having the URI not be connected to the ResponderId. Moreover, I think we can't enforce this inside of this file because it's no STD and FogVIewStoreUri requires std. Let me know what you think.

Reply:
> to having the URI not be connected to the ResponderId

I think with this coming down the line, leaving it as is makes sense.  Though I'm still wondering if pub is the right visibility.  I could see pub(crate), but it seems bad if someone can swap out the responder id mid processing

Reply:
I think it's fine to leave as is but hear what you're saying.